steps/dropbox-upload-v2/0.0.1/step.yml

```diff
@@ -0,0 +1,62 @@
+title: dropbox-upload-v2
```

Comment:
This does not look like a title but id.
Anyway what is the point of ` v2` suffix?

Reply:
Dropbox has released a new API-v2 this "v2" indicates i am using that new API